mapdata/WarcraftLegacies/UnitData/Skin/1093677125.json

```diff
@@ -90,7 +82,7 @@
     {
       "Id": 1651864693,
       "Type": 3,
-      "Value": "Ancient bear hero, adept at frontline melee combat.|n|cfff5962dAbilities:|r Cleaving Attack|n|cfff5962dCan Learn:|R Maiming Strike, War Stomp, Endurance Aura, Avatar|n|n|cffffcc00Attacks land units.|R"
+      "Value": "Ancient bear hero, adept at frontline melee combat.|n|cfff5962dAbilities:|r Heavy Bash|n|cfff5962dCan Learn:|R Maiming Strike, Wild Toughness, Endurance Aura, Avatar|n|n|cffffcc00Attacks land units.|R"
```

Comment:
Why Heavy Bash? Isn't this just normal Bash?

Reply:
It has a much higher bash chance

Reply:
i can change it to just bash

Reply:
Yeah I think we should just go with Bash. Opening up ability renames to simple magnitude differences is a whole can of worms.

Reply:
yup all good, making the changenow!